src/main/frontend/worker/db_worker.cljs

```diff
@@ -313,6 +242,20 @@
   (.exec db "PRAGMA locking_mode=exclusive")
   (.exec db "PRAGMA journal_mode=WAL"))
 
+(defn- gc-sqlite-dbs!
+  "Gc main db weekly and rtc ops db each time when opening it"
+  [sqlite-db client-ops-db datascript-conn {:keys [from-user?]}]
+  (let [last-gc-at (:kv/value (d/entity @datascript-conn :logseq.kv/graph-last-gc-at))]
+    (when (or from-user?
+              (nil? last-gc-at)
+              (not (number? last-gc-at))
+              (> (- (common-util/time-ms) last-gc-at) (* 7 24 3600 1000))) ; 1 week ago
+      (prn :debug "gc current graph")
+      (sqlite-gc/gc-kvs-table! sqlite-db)
+      (d/transact! datascript-conn [{:db/ident :logseq.kv/graph-last-gc-at
+                                     :kv/value (common-util/time-ms)}])))
+  (sqlite-gc/gc-kvs-table! client-ops-db))
```

Comment:
we don't need to handle client-ops-db specially;
just perform GC with the main db at the same frequency